src/main/php/webservices/rest/RestResponse.class.php

```diff
@@ -58,6 +58,30 @@ public function location() {
     return null;
   }
 
+  /**
+   * Returns cookies sent by server
+   *
+   * @see    https://tools.ietf.org/html/rfc6265
+   * @return [:webservices.rest.Cookie]
+   */
+  public function cookies() {
```

Comment:
This method could also return a `webservices.rest.Cookies` instance, which would be iterable, but also have accessor methods to retrieve only valid cookies (for a given path, domain, protocol, etcetera), and would come close to a "CookieJar" functionality exposed by other frameworks.

Reply:
✅ Done!